src/lib/init.ts

```diff
@@ -15,19 +15,56 @@ export type InitProps = {
    * Options for the script loader
    */
   options?: ScriptLoadOptions
+  /**
+   * Shopify International specific options
+   */
+  shopifyInternational?: {
+    language?: string
+    marketId?: string | number
+  }
   /**
    * A custom script loader function to use for loading the Nosto client script
    */
   scriptLoader?: (scriptSrc: string, options?: ScriptLoadOptions) => Promise<void>
 }
 
+function initShopifyInternational({ env, merchantId, shopifyInternational, scriptLoader }: InitProps) {
```

Comment:
Maybe rename the `shopifyInternational` to `shopifyInternationalProps`? The`initShopifyInternational` and `shopifyInternational` feels recursive

Reply:
Maybe rename this method to `initShopifyInternationalScript` or `shopifyInternationalScriptLoader`?